src/UI/Seller/src/app/promotions/components/promotion-edit/promotion-edit.component.html

```diff
@@ -692,24 +692,26 @@ <h6 class="mb-0" translate>ADMIN.PROMOTIONS.USAGE_LIMITS</h6>
           *ngIf="!isCreatingNew"
           buttonText="ADMIN.NAV.PROMOTION"
           (deleteConfirmed)="handleDelete($event)"
+          class="mr-auto ml-n2"
```

Comment:
`.ml-n2` is cheating the `.btn-link` over to the left so it aligns with the card.

Reply:
Super into this approach with the utility class.  Once upon a time I had a global style on the button link, but it wasn't the best solution as it messed with other button links where the alignment wasn't necessary.